src/main/java/com/iexec/common/replicate/ReplicateStatus.java

```diff
@@ -296,6 +337,10 @@ public static List<ReplicateStatus> getMissingStatuses(ReplicateStatus from, Rep
         return statuses.subList(statuses.indexOf(from) + 1, statuses.indexOf(to) + 1);
     }
 
+    /**
+     * @deprecated This unused method will be cleaned.
+     */
+    @Deprecated(forRemoval = true)
     public static List<ReplicateStatus> getSuccessStatusesBeforeComputed() {
```

Comment:
Do we want to remove `isBusyComputing` in `iexec-core` ?

Reply:
I think so. `isBusyComputing` is never used.